ui/src/components/TakeDealDialog.svelte

```diff
@@ -55,25 +54,31 @@
         if (result?.status === 'success') {
           isSuccess = true
           getPeers()
+        } else if (result?.status === 'aborted') {
```

Comment:
could you add the `refunded` case as well?

Reply:
yeah, it happens if the swap fails after funds get locked. maybe you can say something like "swap funds refunded, please check logs for more info"

Reply:
done